scripts/intent_slot/bert_icsl.ipynb

```diff
@@ -0,0 +1,935 @@
+{
```

Comment:
Notebooks are usually committed in the form of `.md` files like https://github.com/dmlc/gluon-nlp/blob/master/docs/examples/machine_translation/gnmt.md which is converted with notedown: https://gluon-nlp.mxnet.io/community/contribute.html#contribute-tutorials